task_processing/plugins/mesos/execution_framework.py

```diff
@@ -138,8 +161,28 @@ def kill_tasks_stuck_in_staging(self):
                         )
                         get_metric(TASK_STUCK_COUNT).count(1)
 
+            self._reconcile_tasks()
             time.sleep(10)
 
+    def _reconcile_tasks(self):
```

Comment:
can you parameterise {{tasks_to_reconcile}} so that we can also use this to run periodic *implicit* reconciliations, as well as these more deliberate explicit reconciliations?

Reply:
Sure.